pkg/github/structs.go

```diff
@@ -151,7 +153,11 @@ func (e CheckSuiteEvent) GetHeadBranch() string {
 	return e.GetCheckSuite().GetHeadBranch()
 }
 
-func (e CheckSuiteEvent) IsValid() bool {
+func (e CheckSuiteEvent) IsValid(c *config.Config) bool {
+	if e.GetCheckSuite().GetApp().GetID() != c.GithubHubAppConfig.App.IntegrationID {
+		log.Debug().Msgf("Discarding event check_suite: not related to this github app")
```

Comment:
maybe our library `github.com/palantir/go-githubapp` can do it automatically ?

Reply:
Did not find yesterday, but I'll check again